src/Form/Type/UserGenderListType.php

```diff
@@ -29,4 +31,27 @@
  */
 class UserGenderListType extends BaseStatusType
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        // NEXT_MAJOR: Call the parent to set the choices
```

Comment:
You can safely remove this NEXT_MAJOR, cause this class is gonna be removed on 5.0